xarray/core/rolling.py

```diff
@@ -814,21 +815,27 @@ def __init__(
         coord_func: str | Callable | Mapping[Any, str | Callable],
     ) -> None:
         """
-        Moving window object.
+        Coarsening object.
 
         Parameters
         ----------
         obj : Dataset or DataArray
             Object to window.
         windows : mapping of hashable to int
-            A mapping from the name of the dimension to create the rolling
-            exponential window along (e.g. `time`) to the size of the moving window.
+            A mapping from the name of the dimension to create the coarsened block along (e.g. `time`) to the size of
+            the coarsened block.
         boundary : {"exact", "trim", "pad"}
             If 'exact', a ValueError will be raised if dimension size is not a
             multiple of window size. If 'trim', the excess indexes are trimmed.
             If 'pad', NA will be padded.
         side : 'left' or 'right' or mapping from dimension to 'left' or 'right'
-        coord_func : function (name) or mapping from coordinate name to function (name).
+        coord_func : function, str name of function, or mapping from coordinate name to function or str name of func.
```

Comment:
```suggestion
        coord_func : Callable, str name of function, or mapping from coordinate name to Callable or str name of func.
```
Also probably more readable that way because you don't have 4 times "function" in one sentence.

Reply:
I think we should add an explicit list  of function names